src/main/java/com/eternalcode/core/EternalCore.java

```diff
@@ -93,49 +96,54 @@ public void onEnable() {
         // Services
         this.userService = new UserService();
         this.funnyCommands = FunnyCommands.configuration(() -> this)
-            .bind(resources -> resources.on(EternalCore.class).assignInstance(this))
-            .bind(resources -> resources.on(UserService.class).assignInstance(userService))
-            .bind(resources -> resources.on(ConfigurationManager.class).assignInstance(configurationManager))
-            .bind(resources -> resources.on(ScoreboardManager.class).assignInstance(scoreboardManager))
-            .bind(new CommandInfoBind())
-            .registerDefaultComponents()
-            .permissionHandler((message, permission) -> message.getCommandSender().sendMessage(ChatUtils.color(config.permissionMessage.replace("{PERMISSION}", permission))))
-            .commands(
-                AlertCommand.class,
-                AnvilCommand.class,
-                CartographyTableCommand.class,
-                ChatCommand.class,
-                ClearCommand.class,
-                DisposalCommand.class,
-                EnderchestCommand.class,
-                FeedCommand.class,
-                FlyCommand.class,
-                GamemodeCommand.class,
-                GcCommand.class,
-                GodCommand.class,
-                GrindstoneCommand.class,
-                HatCommand.class,
-                HealCommand.class,
-                KillCommand.class,
-                SkullCommand.class,
-                SpeedCommand.class,
-                StonecutterCommand.class,
-                WhoIsCommand.class,
-                WorkbenchCommand.class,
-                EternalCoreCommand.class,
-                ScoreboardCommand.class,
-                AdminChatCommand.class,
-                HelpOpCommand.class
-            ).install();
+                .bind(resources -> resources.on(EternalCore.class).assignInstance(this))
```

Comment:
Faktycznie zawsze się robiło na 2xtab chaina...
Przydałoby się to jakoś ujednolicić, bo w projekcie eternala automatycznie daje (mi) 1xtab.
Nie wiem, czy to przez jakieś ustawienia projektu, czy IJ samo mi to ustawiło, patrząc na kontekst...

Reply:
Już to ogarnąłem, klikłem Rearrange code, przez przypadek i się zbugowało, po ponownym użyciu funkcji działa tak jak było wczęśniej 👍